commands/command_migrate.go

```diff
@@ -151,17 +151,24 @@ func includeExcludeRefs(l *tasklog.Logger, args []string) (include, exclude []st
 			include = append(include, ref.Refspec())
 		}
 	} else {
-		// Otherwise, if neither --include-ref=<ref> or
-		// --exclude-ref=<ref> were given, include no additional
-		// references, and exclude all remote references that are remote
-		// branches or remote tags.
-		remoteRefs, err := getRemoteRefs(l)
+		bare, err := git.IsBare()
```

Comment:
Would it make sense to do this inside `getRemoteRefs()`? If it returns true, then return a nil `remoteRefs` slice. Not a big deal, but I think it could improve the flow in this code a tad.

Reply:
Good call, I could go either way on this. I like that `getRemoteRefs()` _always_ returns remote refs, which to me feels "true" even in the bare case, since the remote refs _exist_, they just don't carry any meaning. I like pushing the decision of "do we care about remote refs" out of `getRemoteRefs()` itself.